### PR TITLE
Dz 3

### PR DESCRIPTION
**Pull request by Nazarov-AA**
DZ_3 DONE

### CODE REVIEW

File: app/src/main/java/com/example/dz/MyVM2.kt [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import kotlin.random.Random
+
+
+class MyVM2 : ViewModel() {
+
+    var potatoes1 = MutableLiveData(0)
```
Comment (stasenkots) [inline app/src/main/java/com/example/dz/MyVM2.kt:16]: Create class to keep potatoes value,cabbage value and so on

**Review by stasenkots — CHANGES_REQUESTED** [review]

**Review by Plehanov-K — CHANGES_REQUESTED** [review]
Новый точечный рисунок


Button "Go" is not visible.

File: app/src/main/res/layout/activity_main6.xml [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        android:layout_margin="16dp"
+        android:background="#BEEBF1">
+
+
+        <TextView
+            android:id="@+id/textView0"
+            android:layout_margin="8dp"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="Минская область"
```
Comment (Plehanov-K) [inline app/src/main/res/layout/activity_main6.xml]: android:text="**Минская область**"  Some hardcode

File: app/src/main/java/com/example/dz/MyVM2.kt (hunk 1 above)
Comment (Nazarov-AA) [reply app/src/main/java/com/example/dz/MyVM2.kt:16]: next time I'll do better)) 
this time the customer is state and does not demand more)

Comment (TMSlebedev) [reply app/src/main/java/com/example/dz/MyVM2.kt:16]: =))

**Review by Plehanov-K — APPROVED** [review]

**Review by stasenkots — APPROVED** [review]

**Review by TMSlebedev — APPROVED** [review]

**Pull request merged**